src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/Microsoft.NET.Workload.Mono.Toolchain.Manifest.pkgproj

```diff
@@ -6,12 +6,30 @@
   </PropertyGroup>
 
   <Target Name="_PrepareForPack" BeforeTargets="GetPackageFiles" Returns="@(PackageFile)">
+    <!-- Override the id to include the sdk band as per the workload installer spec -->
+    <PropertyGroup>
+      <SdkBandVersion>6.0.100</SdkBandVersion>
```

Comment:
Should the version be hardcoded here?

Reply:
I think so, temporarily.

It's my understanding that when the sdk band bumps (say to 6.0.200), we will need to publish a new baseline package.  I'm wondering if we need to place this in a new repo for that reason alone?  That is, unless we release a new runtime with each sdk bump.

Id like to get the right package out to the sdk now and shift to a new repo, if necessary, shortly after.

Reply:
If the file stays here I think the version should go to runtime/eng/Versions.props

Reply:
Sounds good to me.